src/app/zero-shot-details/component/zero-shot-details.component.ts

```diff
@@ -208,6 +209,7 @@ export class ZeroShotDetailsComponent
       this.informationSource = informationSource;
       this.description = informationSource.description;
       this.informationSourceName = informationSource.name;
+      this.lastTaskDisplay = parseUTC(this.informationSource.lastTask.createdAt);
```

Comment:
i'd change it to be part of the task. Also this should be part of the if statement since lastTask doesn't always exist